gapic-generator-java/src/main/java/com/google/api/generator/gapic/composer/common/AbstractTransportServiceStubClassComposer.java

```diff
@@ -190,6 +192,20 @@ public GapicClass generate(GapicContext context, Service service) {
             messageTypes,
             context.restNumericEnumsEnabled());
 
+    List<MethodDefinition> methodDefinitions =
```

Comment:
Can we move this HttpJson/REST specific logic to `HttpJsonServiceStubClassComposer` instead of in this abstract composer?